gramps/gui/selectors/baseselector.py

```diff
@@ -318,12 +318,6 @@ def column_clicked(self, obj, data):
             self.model.reverse_order()
         self.build_tree()
 
-        handle = self.first_selected()
-        if handle:
-            path = self.model.on_get_path(handle)
```

Comment:
You could keep the current handle selected and visible with:
```
iter_ = self.model.get_iter_from_handle(handle)
path = self.model.get_path(iter_)
```

Reply:
Removing all these lines change nothing.
The selected lines is always selected after sorting on another column.
As I said in the bug report, I think it was a workaround for an old Gtk problem.

I'll try what you say and see if something changes.

Reply:
OK.  I it works anyway it is better to remove the code.

Reply:
Tried and the result is the same. I think we can remove these lines.

Reply:
Thanks for checking.  I'll merge this now.